flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -446,7 +546,11 @@ private static BufferResponse createBufferResponse(
 		// Deserialize the bytes again. We have to go this way, because we only partly deserialize
 		// the header of the response and wait for a buffer from the buffer pool to copy the payload
 		// data into.
```

Comment:
`wait for a buffer from the buffer pool to copy the payload data into.` should be properly adjusted?

Reply:
Fixed the comments.